pkg/clusterfile/clusterfile.go

```diff
@@ -173,13 +174,13 @@ func (c *ClusterFile) SaveAll(opts SaveOptions) error {
 	}
 
 	if opts.CommitToCluster {
-		return saveToCluster(content)
+		return saveToCluster(content, opts.ConfPath)
 	}
 	return nil
 }
 
-func saveToCluster(data []byte) error {
-	cli, err := kubernetes.GetClientFromConfig(kubernetes.AdminKubeConfPath)
+func saveToCluster(data []byte, confPath string) error {
```

Comment:
We can try to optimize this function: after create a new cluster, we got a runtime driver to do it.

Reply:
Ok, I would like to try to optimize it next time, also include k8s package logic.